app/views/presentations/_filter.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_tag presentations_path, method: 'get' do %>
+  <p>
+    <%= label_tag('Filter by Presenter:') %>
+    <%= collection_select(:presentation, :person_id, Person.all, :id, :first_name, prompt: true) %>
```

Comment:
I've added in `include_blank: 'All Presenters' which allows us to filter for All presenters in the controller.